targets/python/PythonSDK/PythonSDK.pyproj

```diff
@@ -0,0 +1,35 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="4.0">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>49e5c759-a723-4d9e-b806-9cf9035c5e0a</ProjectGuid>
+    <ProjectHome>.</ProjectHome>
+    <StartupFile>PythonSDK.py</StartupFile>
```

Comment:
References empty file.
Maybe this should be an example which executes something.